src/main/java/racingcar/v1/domain/RacingGame.java

```diff
@@ -0,0 +1,68 @@
+package racingcar.v1.domain;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+import static java.util.stream.Collectors.toList;
+
+public class RacingGame {
+
+    private final int MOVE_FORWARD_CONDITION = 4;
+    private final int MOVE_UNIT = 1;
+    private int rotationLeft;
+    private List<Car> carList;
+
+    public RacingGame(String carNames, int rotation) {
+        this.carList = parseCarListFromString(carNames);
+        this.rotationLeft = rotation;
+    }
+
+    public List<Car> parseCarListFromString(String cars) {
+        return Arrays.stream(cars.split(","))
+                .map(name -> new Car(name.trim()))
+                .collect(Collectors.toList());
+    }
+
+    public List<Car> getCarList() {
+        return this.carList;
+    }
+
+    public boolean isEnd() {
+        return this.rotationLeft <= 0;
+    }
+
+    public void playSingleLoop() {
```

Comment:
<img width="991" alt="스크린샷 2023-04-16 오후 11 06 43" src="https://user-images.githubusercontent.com/12059592/232317284-98cf590a-2031-4660-8217-3aa84f013622.png">


```
interface 숫자생성기() {
  숫자생성()
}
```
```suggestion
    public void playSingleLoop(숫자생성기 ) {
```

다양한 방법이 있겠지만, 추상화 방식을 사용해서 예측가능한 테스트를 진행할수 있도록 해볼수 있을것같아요!